### PR TITLE
UIOR-20 removed api calls since they've been removed on back-end

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,18 +27,14 @@
       "eresource": "1.0",
       "license": "1.0",
       "location": "1.0",
-      "order_format": "1.0",
-      "order_type": "1.0",
       "payment_status": "1.0",
```
Comment (craigmcnally) [inline package.json]: payment_status is gone too, but the mod-orders-storage module descriptor incorrectly still has it listed.  @komick can you please fix the mod-orders-storage MD?

File: package.json [hunk 2]
```diff
@@ -178,13 +178,9 @@
...
           "license.all",
           "location.all",
-          "order_format.all",
-          "order_type.all",
           "physical.all",
-          "receipt_status.all",
           "renewal.all",
-          "source.all",
-          "workflow_status.all"
+          "source.all"
```
Comment (craigmcnally) [inline package.json]: reporting_code.all?

Comment (aliaksei-chumakou) [reply package.json]: @craigmcnally do you mean we need to add this line? I believe it could be done with actual usage of this later

Comment (craigmcnally) [reply package.json]: I wasn't sure.  If we don't need it now adding it later is fine

**Pull request merged**